main.js

```diff
@@ -105,252 +135,361 @@ var ArmorDfs = {
+    "Pacifist credits [Dogchecked]",
+    "Pacifist credits - Highway [Dogchecked]",
+    "Pacifist credits - Beach [Dogchecked]",
+    "Pacifist credits - Mettaton performance [Dogchecked]",
+    "Pacifist credits - School entrance [Dogchecked]",
+    "Pacifist credits - Mount Ebott [Dogchecked]",
+    "Pacifist credits - Special Thanks [Dogchecked]",
+    "Pacifist credits - Bedroom [Dogchecked]",
+    "Pacifist credits - The End [Dogchecked]",
+    "Sprite Test [Dogchecked]",
+    "Settings menu [Dogchecked]",
+    "Control test room [Dogchecked]",
+    "Omega Flowey - Initializer [Dogchecked]",
+    "Omega Flowey - Fake intro [Dogchecked]",
+    "Omega Flowey - Fake menu [Dogchecked]",
+    "Omega Flowey - Fake save point [Dogchecked] [SAVE?]",
+    "Omega Flowey [Dogchecked]",
+    "Omega Flowey - Respawn mocking [Dogchecked]",
+    "room_fire4 [Dogchecked]",
+    "room_fire10_old [Dogchecked]",
+    "room_fire10A_old [Dogchecked]",
+    "Snowdin - Ice Puzzle [Dogchecked]",
+    "Ruins - Rock Candy [Dogchecked]",
+    "Snowdin - Growing snowball [Dogchecked]",
+    "Waterfall - Torch puzzle [Dogchecked]",
+    "Waterfall - Meet Undyne (Unused) [Dogchecked]",
+    "Waterfall - Rooted mushroom [Dogchecked]",
+    "room_monsteralign_test [Dogchecked]",
+    "Battle room [Dogchecked]",
+    "Flowey battle room [Dogchecked]",
+    "\"Fast\" battle room [Dogchecked]",
+    "New Home - Story battle room [Dogchecked]",
+    "Game Over [Dogchecked]",
+    "Snowdin Town - Shop [Dogchecked]",
+    "Waterfall - Gerson's shop [Dogchecked]",
+    "Hotland - Bratty and Catty [Dogchecked]",
+    "Hotland - Burgerpants [Dogchecked]",
+    "Waterfall - Tem Shop [Dogchecked]",
+    "River person transition room [Dogchecked]",
+    "Papyrus date [Dogchecked]",
+    "Alphys date [Dogchecked]",
+    "Omega Flowey - Wilting [Dogchecked]",
+    "Flowey's pacifist speech [Dogchecked]",
+    "room_empty [Dogchecked]",
+    "room_emptywhite [Dogchecked]",
+    "room_emptyblack [Dogchecked]",
+    "Genocide - The Nothingness [Dogchecked]",
+    "Snowdin - Title [Dogchecked]",
+    "Dogcheck room",
+    "Hotland - Quiz battle (Unused) [Dogchecked]",
+    "Pre-Asriel scene (Unused) [Dogchecked]",
+    "Pre-Asriel scene 2 (Unused) [Dogchecked]",
+    "Asriel appears [Dogchecked]",
+    "Asriel room [Dogchecked]",
+    "Asriel flashbacks [Dogchecked]",
+    "Asriel animation test [Dogchecked]",
+    "Asriel phase 2 animation test [Dogchecked]",
+    "Snowdin - Dog Shrine [Dogchecked]"
+];
+
+for (var i = 0; i < rooms[2].length; i++) {
+    if (rooms[2][i].indexOf("[SAVE]") !== -1) {
+        rooms[0][i] = rooms[2][i];
+        rooms[1][i] = rooms[2][i];
+    } else if (rooms[2][i].indexOf("[Dogchecked]") === -1) {
+        rooms[1][i] = rooms[2][i];
+    }
+}
+// rooms[0] is SAVE points only, rooms[1] is non-Dogchecked only, rooms[2] contains all rooms.
+
+var roomSelectOptions = [
+    "SAVE points only",
+    "Accessible rooms only",
+    "All rooms"
+];
```

Comment:
I think accessible rooms should remain the default option so the new option doesn't confuse previous users.